widget_driver/README.md

```diff
@@ -52,6 +52,17 @@ Just focus on the current widget under test, and forget about all other child wi
 
 Let's get started and create our first `WidgetDriver`!
 
+### 1: Add a dependency to WidgetDriver
+
+In your project add a dependency to widget driver in your `pubspec.yaml` and run `flutter pub get`
+
```

Comment:
We already have a section in the readme about how to install/add the dependency
https://github.com/bmw-tech/widget_driver/tree/master/widget_driver#installation

But maybe it is better to have it up-front up here also.
But I would then not add a specific version number in the example.
I would write as in the install-section.
Something like this:
```
widget_driver: <latest_version>
```
and then maybe a link down to the install section.

Plus if you really want to use widget driver, then you also need the generator.. sort of.. and the builder..
And they are both mentioned in the install section

Reply:
Instead of referencing to the installation section here, I just decided to move the INSTALLATION section now right before USAGE (it was way deeper in the document) - this way it should also be clear.